Sources/PostgresNIO/New/PSQLChannelHandler.swift

```diff
@@ -18,6 +18,9 @@ final class PSQLChannelHandler: ChannelDuplexHandler {
             self.logger.trace("Connection state changed", metadata: [.connectionState: "\(self.state)"])
         }
     }
+    
+    #warning("This is an ugly hack, is there some way around it?")
+    private var handlerContext: ChannelHandlerContext?
```

Comment:
@weissi How bad is it to store a `ChannelHandlerContext` for use later? With this property we can use it once we get an event that is not triggered by the ChannelPipeline.

Reply:
It’s fine. See below